src/components/ConnectModal.tsx

```diff
@@ -11,6 +11,29 @@ interface ConnectModalProps {
   closeModal: () => void;
 }
 
+const connectEmail = async () => {
+  const msg = {
+    to: "devashishsood18@gmail.com", // Replace with your recipient
+    from: "devashishsood9@gmail.com", // Replace with your verified sender
+    subject: "New Contact Message",
+    text: `TESTTESTTEST`,
+  };
+
+  // const objToQuery = (obj: {[key: string]: string}): string => {
+  //   return Object.keys(obj).map(key => `${key}=${obj[key]}`).join("&")
+  // }
+
+  // ? key=value&key2=value2
+
+  const result = await fetch(`/api/sendEmail`, {
```

Comment:
Why don't we make this a tRPC endpoint?

Reply:
That is what it will be once I can work with it using the new changes on main